docs/functions/public.rst

```diff
@@ -49,10 +49,19 @@ their config.
    The optional flag ``--list`` will instead list all currently loaded
    themes.
 
+   The variable :envvar:`LP_THEME` can be set to a theme name which functions
+   like calling this function. :envvar:`LP_THEME` will always hold the
```

Comment:
The first sentence is a bit hard to grasp. What about: "The variable LP_THEME can be set to a theme name, which is essentially equivalent to calling this function."